iocore/cache/I_CacheDefs.h

```diff
@@ -114,8 +114,7 @@ enum CacheDataType {
 
 enum CacheFragType {
   CACHE_FRAG_TYPE_NONE,
-  CACHE_FRAG_TYPE_HTTP_V23, ///< DB version 23 or prior.
-  CACHE_FRAG_TYPE_RTSP,     ///< Should be removed once Cache Toolkit is implemented.
+  CACHE_FRAG_TYPE_RTSP, ///< Should be removed once Cache Toolkit is implemented.
   CACHE_FRAG_TYPE_HTTP,
```

Comment:
Doesn't this change the value of `CACHE_FRAG_TYPE_HTTP`? That would seem to be a problem if the disk image has a fragment type of `4` (the current value) but the code expects a value of `2`.

Reply:
You are right. That's would be a horrible.

Reply:
So let's keep this CACHE_FRAG_TYPE_HTTP_V23 for not.

Reply:
Another option is of course to remove the old enum symbols, and code the values for the remaining ones accordingly. I’m ok either way.

Reply:
I would keep the enum and use `== 4`. It's possible we'll have other types in the future.

Reply:
You might put in `CACHE_FRAG_TYPE_UNUSED_1`, etc., to occupy the space as well, just to be very clear.